.gitlab-ci.yml

```diff
@@ -210,7 +210,7 @@ test-build-linux-stable:
     - echo "Polkadot version = ${VERSION} (EXTRATAG = ${EXTRATAG})"
     - echo -n ${VERSION} > ./artifacts/VERSION
     - echo -n ${EXTRATAG} > ./artifacts/EXTRATAG
-    - cp -r scripts/docker/* ./artifacts
+    - cp scripts/docker/* ./artifacts
```

Comment:
Are you sure about that one ?
I see we have already the staking miner images under [its own folder](https://github.com/paritytech/polkadot/tree/master/scripts/docker/staking-miner).

Since we expect to have at least 2 images (injected + builder) pro binary, we will likely have only folders under `scripts/docker` so the `-r` will become useful.

Reply:
>Are you sure about that one ?
Yes and no :)
So currently in .gitlab-ci.yml we use only files in scripts/docker/ folder (excluding subfolders). I wasn't sure about staking-miner but I couldn't find any steps related to it. But I don't know about future intensions.
Also need @TriplEight opinion.

Reply:
The staking-miner is rather new, so are its images. But those should be built/published as well just like polkadot:
- simple test build of the 'builder' image
- build + push on release for the 'injected' image